tests/openwisp2/settings.py

```diff
@@ -25,6 +25,20 @@
     'PORT': '8086',
 }
 
+INFLUXDB_SETTINGS = {
```

Comment:
why is this called `INFLUXDB_SETTINGS`?

Reply:
Ok, answering for this one and below question too. I thought it might be better to have two data structure named `INFLUXDB_SETTINGS` and `ELASTICSEARCH_SETTINGS`. `TIMESERIES_DATABASE` would be one of the two (defaulting to `INFLUXDB`) depending on whether the environment variable `elasticsearch` is set or not. This would help to run tests for both the databases at the same time. Check `.travis.yml` and you will get a better idea of how it's working. This has been added purely for testing purposes, the users do not need to define it.

If you have a better idea of how this can be done then let me know, thanks!

Reply:
this is called INFLUXDB_SETTINGS but it loads elasticsearch, why? Doesn't make sense to me